### PR TITLE
Make ssIndices work with ssLookupFile and trailing slashes

### PR DESCRIPTION
**Pull request by soenkehahn**

### CODE REVIEW

File: wai-app-static/test/WaiAppEmbeddedTest.hs [hunk 1]
```diff
@@ -32,7 +32,7 @@ embSpec = do
                              { requestHeaders = [("If-None-Match", "Etag 1")] }
             assertStatus 304 req
 
-        it "ssIndices works" $ do
+        it "ssIndices work" $ do
```
Comment (snoyberg) [inline wai-app-static/test/WaiAppEmbeddedTest.hs]: Why did you change this? I think the former grammar makes more sense (`ssIndices` is a singular noun as its own configuration value)

**Comment by snoyberg** [comment]
Minor grammar comment, but not worth holding up the PR on it. Thanks!

File: wai-app-static/test/WaiAppEmbeddedTest.hs (hunk 1 above)
Comment (soenkehahn) [inline wai-app-static/test/WaiAppEmbeddedTest.hs]: I thought it sounded better. But here's the reverting PR: https://github.com/yesodweb/wai/pull/574

**Comment by soenkehahn** [comment]
Thanks for merging!
